tests/cpp/test_gpu_tensorcore.cpp

```diff
@@ -2482,6 +2482,19 @@ TEST_F(
           params.double_buffer_options.smem_double_buffer_stage,
           data_types,
           /*ignore_occupancy_drop=*/false);
+
+  // Find GPU name to check if this is an A100
+  int dev_idx = 0;
+  NVFUSER_CUDA_RT_SAFE_CALL(cudaGetDevice(&dev_idx));
+  cudaDeviceProp prop;
+  NVFUSER_CUDA_RT_SAFE_CALL(cudaGetDeviceProperties(&prop, dev_idx));
+  std::string gpu_name(prop.name);
+  if (gpu_name.find("A100") != gpu_name.npos) {
```

Comment:
Are prop.major and prop.minor more reliable than names? 

https://docs.nvidia.com/cuda/cuda-runtime-api/structcudaDeviceProp.html#structcudaDeviceProp_164490976c8e07e028a8f1ce1f5cd42d6

Reply:
You're right. I was thinking I needed to be more specific, but every 8.0 device supports the same max smem per cta.

Reply:
Pushed change that uses `deviceMajorMinorCheck` instead of this.